daemon/src/projection.rs

```diff
@@ -729,6 +730,40 @@ impl Cfd {
 
         Some(url)
     }
+
+    pub fn is_open(&self) -> bool {
+        let in_open_state = vec![
+            CfdState::PendingOpen,
+            CfdState::Open,
+            CfdState::PendingCommit,
+            CfdState::OpenCommitted,
+            CfdState::IncomingRolloverProposal,
+            CfdState::IncomingSettlementProposal,
+            CfdState::OutgoingRolloverProposal,
+            CfdState::OutgoingSettlementProposal,
+            CfdState::RolloverSetup,
+        ]
```

Comment:
Not a big fan of this at all. These states only exist because they were originally exposed on the API and used for the internal model. They _only_ exist in the projection layer for backwards-compatibility reasons.

I think we should take the opportunity when exposing first-class metrics from the daemon and reason about the desired state based on the actual source-of-truth which are the events.